common/data/okn/oknObject.ts

```diff
@@ -19,7 +19,7 @@ export interface OknAttributes {
     date: string;
     type: string;
     category: OknObjectSignificanceType;
-    img?: string;
+    img?: any;
```

Comment:
Images have type { url: string; title: string }. That's what I see in okn objects in strapi